src/message_interface/response.rs

```diff
@@ -109,6 +109,10 @@ pub enum Response {
     GeneratedMnemonic(String),
     /// Response for [`GetNodeInfo`](crate::message_interface::Message::GetNodeInfo)
     NodeInfo(NodeInfoWrapper),
+    /// Response for [`Bech32ToHex`](crate::message_interface::Message::Bech32ToHex)
+    Bech32ToHex(String),
+    /// Response for [`HexToBech32`](crate::message_interface::Message::HexToBech32)
+    HexToBech32(String),
```

Comment:
How about calling it `HexAddress` and `Bech32Address`? I think this would be clearer

Reply:
I agree, will change.